master/pyext/src/validation/__init__.py

```diff
@@ -16,6 +16,42 @@
 from collections import defaultdict
 from validation import utility
 
+import logging
+from typing import Final
+
+#########################
+# Setup operational mode
+#########################
+
+logging.basicConfig(level=logging.INFO)
+IHMV_MODES: Final = ["PRODUCTION", "DEVELOPMENT"]  # Available modes
+
+
+def get_operational_mode() -> str:
+    """ Check environment variables and set the operational mode"""
+    ihmv_env: str = "PRODUCTION"  # Default mode
+    ihmv_env_name = "IHMV_MODE"  # Name of the environment variable
+    ihmv_env_ = os.environ.get(ihmv_env_name)  # Check the environment variable
+
+    if (ihmv_env_ is not None) and (ihmv_env_ in IHMV_MODES):
```

Comment:
This seems unnecessarily verbose, and the parentheses are redundant. `if ihmv_env_ in IHMV_MODES:` would do the job.